docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -246,3 +246,106 @@ function hs_paragraph_types_update_10006() {
     }
   }
 }
+
+/**
+ * Set new config for collection color wells.
+ */
+function hs_paragraph_types_update_10007(&$sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  if (empty($sandbox['ids'])) {
+
+    // Import all config before updating existing paragraphs.
+    _hs_paragraph_types_well_color_config();
```

Comment:
Instead of running a separate function, I'd put this in its own database update hook that runs before this one. I.e., Create the necessary configuration in the first update hook, then in the second update hook do anything data related.

Reply:
Fixed!